daprdocs/content/en/concepts/overview.md

```diff
@@ -43,84 +43,78 @@ Each of these building blocks is independent, meaning that you can use one, some
+### Dapr SDKs
 
-## Designed for Operations
-Dapr is designed for [operations](/operations/). The [services dashboard](https://github.com/dapr/dashboard), installed via the Dapr CLI, provides a web-based UI enabling you to see information, view logs and more for the Dapr sidecars.
+To make using Dapr more natural for different languages, it also includes [language specific SDKs]({{<ref sdks>}}) for:
+- C++
+- Go
+- Java
+- JavaScript
+- Python
+- Rust
+- .NET
+- PHP.
 
-The [monitoring tools support](/operations/monitoring/) provides deeper visibility into the Dapr system services and side-cars and the [observability capabilities]({{<ref "observability-concept.md">}}) of Dapr provide insights into your application such as tracing and metrics.
+These SDKs expose the functionality of the Dapr building blocks through a typed language API, rather than calling the http/gRPC API. This enables you to write a combination of stateless and stateful functions and actors all in the language of their choice. And because these SDKs share the Dapr runtime, you get cross-language actor and functions support.
 
-## Run anywhere
+### Developer frameworks
 
-### Running Dapr on a local developer machine in self hosted mode
+Dapr can be used from any developer framework. Here are some that have been integrated with Dapr:
 
-Dapr can be configured to run on your local developer machine in [self-hosted mode]({{< ref self-hosted >}}). Each running service has a Dapr runtime process (or sidecar) which is configured to use state stores, pub/sub, binding components and the other building blocks. 
+#### Web
 
-You can use the [Dapr CLI](https://github.com/dapr/cli#launch-dapr-and-your-app) to run a Dapr enabled application on your local machine. Try this out with the [getting started samples]({{< ref getting-started >}}). 
+| Language | Frameworks | Description |
+|----------|------------|-------------|
+| [.NET]({{< ref dotnet >}}) | [ASP.NET]({{< ref dotnet-aspnet.md >}}) | Brings stateful routing controllers that respond to pub/sub events from other services. Can also take advantage of [ASP.NET Core gRPC Services](https://docs.microsoft.com/en-us/aspnet/core/grpc/).
+| [Java](https://github.com/dapr/java-sdk) | [Spring Boot](https://spring.io/) 
+| [Python]({{< ref python >}}) | [Flask]({{< ref python-flask.md >}})
+| [Javascript](https://github.com/dapr/js-sdk) | [Express](http://expressjs.com/)
+| [PHP]({{< ref php >}}) | | You can serve with Apache, Nginx, or Caddyserver.
 
-<img src="/images/overview_standalone.png" width=800>
+#### Integrations
 
-### Running Dapr in Kubernetes mode
+Visit the [integrations]({{< ref integrations >}}) page to learn about some of the first-class support Dapr has for various frameworks and external products, including:
+- Visual Studio Code
```

Comment:
This is not really an integration VSCode. I would remove. And if we are going to have a list here (which may get a bit long) we need to add Azure API Management (APIM) as one of the more useful integrations. I would add this for now and we need to address the issue for this at some point

Reply:
Idk, I'd still call the VSCode extension an "integration". We integrate Dapr building blocks, dubugging, and more into the VSCode experience. I'll update the header to say "Integrations and extensions"